packages/config/config/devices/0x0131/ph-pse02.json

```
@@ -0,0 +1,31 @@
+{
+	"manufacturer": "Zipato",
+	"manufacturerId": "0x0131",
+	"label": "PH-PSE02",
+	"description": "Multisound indoor siren",
```

```suggestion
	"description": "Multisound Indoor Siren",
```
https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide?id=device-descriptions
> Device descriptions should be **Title Case**.